src/qml/NightstandPage.qml

```diff
@@ -116,71 +143,60 @@ Item {
                 }
             }
 
+            Item { width: parent.width; height: Dims.h(6) }
+
             Item {
                 width: parent.width
-                height: Dims.l(20)
-
-                MouseArea {
-                    anchors.fill: parent
-                    hoverEnabled: true
-                    onClicked: nightstandUseCustomWatchface.value = !nightstandUseCustomWatchface.value
-                    Rectangle {
-                        anchors.fill: parent
-                        color: "white"
-                        opacity: parent.containsPress ? 0.2 : 0
-                    }
-                }
+                height: rowHeight
 
                 LabeledSwitch {
                     anchors.fill: parent
+                    height: rowHeight
                     //% "Custom watchface"
                     text: qsTrId("id-nightstand-custom-watchface")
                     checked: nightstandUseCustomWatchface.value
                     opacity: nightstandEnabled.value ? 1.0 : 0.4
                     onCheckedChanged: nightstandUseCustomWatchface.value = checked
+                    enabled: nightstandEnabled.value
+
+                    Behavior on opacity {
```

Comment:
Not blocking for this PR, but adding this to the `LabeledSwitch` component might be better?